src/cdi_eospac/Eospac.cc

```diff
@@ -340,12 +340,12 @@ std::vector<double> Eospac::getF(std::vector<double> const &vdensity,
 
   // There is one piece of returned information for each (density, temperature) tuple.
   Check(vtemperature.size() < INT32_MAX);
-  int returnSize = static_cast<int>(vtemperature.size());
+  EOS_INTEGER returnSize = static_cast<int>(vtemperature.size());
```

Comment:
Should this be `static_cast<EOS_INTEGER>`? Maybe I'm missing something

Reply:
Fixed.